rest-api/test/unit_test/api_test/participant_summary_api_test.py

```diff
@@ -249,6 +252,115 @@ def testQuery_noSummaries(self):
+    response = self.send_get(url)
+    # We have the same number of participants as summaries
+    self.assertEqual(len(response['entry']), len(first_batch))
+    last_modified_list = list()
+    first_batch_list = list()
+    for i in response['entry']:
+      last_modified_list.append(i['resource']['lastModified'])
+    for i in first_batch:
+      first_batch_list.append(i['lastModified'])
+
+    self.assertListEqual(last_modified_list, sorted(first_batch_list))
+    # With the same ID's (they're the same participants)
+    self.assertEqual(
+      sorted([p['participantId'] for p in first_batch]),
+      sorted([p['resource']['participantId'] for p in response['entry']]),
+    )
+    self.assertEqual(sorted([p['resource']['lastModified'] for p in response['entry']]),
+                      [p['resource']['lastModified'] for p in response['entry']])
+
+    # Get the next chunk with the sync url
+    # Verify that this is, in fact, a sync URL - not a next
+    sync_url = response['link'][0]['url']
+    index = sync_url.find('ParticipantSummary')
+    self.assertEqual(response['link'][0]['relation'], 'sync')
+
+    # Verify that the next chunk has no results (everyone is accounted for)
+    response2 = self.send_get(sync_url[index:])
+    self.assertEqual(len(response2['entry']), 0)
+    self.assertEqual(response2['entry'], [])
+
+    # Create a second batch
+    second_batch = [setup_participant(t4) for _ in range(10)]
+    response3 = self.send_get(sync_url[index:])
+    # We have the same number of participants as summaries
+    self.assertEqual(len(response3['entry']), len(second_batch))
+    # With the same ID's (they're the same participants)
+    self.assertEqual(
+      sorted([p['participantId'] for p in second_batch]),
+      sorted([p['resource']['participantId'] for p in response3['entry']]),
+    )
+
+    no_count_url = 'ParticipantSummary?lastModified=lt%s&_sync=true&awardee=PITT' % TIME_4
+    no_count_response = self.send_get(no_count_url)
+    total_count = len(no_count_response['entry'])
+    self.assertEquals(total_count, 20)
+    url = 'ParticipantSummary?lastModified=lt%s&_count=10&_sync=true&awardee=PITT' % TIME_4
+    response = self.send_get(url)
+    self.assertEqual(len(response['entry']), 10)
+    next_url = response['link'][0]['url']
+    next_10 = self.send_get(next_url[index:])
+    self.assertEqual(len(next_10['entry']), 10)
+
+    no_last_modified_url = 'ParticipantSummary?_sync=true&awardee=PITT'
+    no_lm_response = self.send_get(no_last_modified_url)
+    self.assertEquals(len(no_lm_response['entry']), 20)
+    self.assertEquals(no_lm_response['link'][0]['relation'], 'sync')
+    # ensure same participants are returned before 5 min. buffer
+    sync_url = no_lm_response['link'][0]['url']
+    setup_participant(t4)
+    add_one_more = self.send_get(sync_url[index:])
+    print len(add_one_more['entry'])
+    # self.assertEquals(len(add_one_more['entry']), 21)
```

Comment:
I would have expected this to return some of the results from before (anything modified in the last 60 seconds) plus the new one.